src/components/editor.rs

```diff
@@ -2892,6 +2904,109 @@ pub(crate) enum DispatchEditor {
     Dedent,
 }
 
+impl std::fmt::Display for DispatchEditor {
```

Comment:
I'm not sure if we should use a proc_macro for this or keep it that way. When using debug printing it will display other information alongside the enum variant name (i.e. parameters like `use_system_clipboard`) which might not be desired, as it will fill up the global bar.

Reply:
I think this code is fine for now, but eventually, I want to have an extra line for the status bar, so we can show the full dispatch details in the second line without having to be afraid that it will overflow.

Reply:
There might be need to overhaul the bar as a hole later anyway, where we can change this too, as advanced user might want to disable this feature anyway. Do you want to keep this in mind or create another issue for later?